dashboard-widgets.el

```diff
@@ -302,26 +303,23 @@ Optional argument REP is the replacement string of non-displayable character."
   "Get the formatted icon for DIR.
 ARGS should be a plist containing `:height', `:v-adjust',
 or `:face' properties."
-  (dashboard-replace-displayable
    (pcase dashboard-icon-type
      ('all-the-icons (apply #'all-the-icons-icon-for-dir dir args))
-     ('nerd-icons (apply #'nerd-icons-icon-for-dir dir args)))))
+     ('nerd-icons (nerd-icons-icon-for-dir dir args))))
```

Comment:
Why are we removing `apply` here? 🤔

Reply:
As far as I know `apply` does not work with macros. In my setup it wasn't working.